runtime-infra/fragments/ecs-service.yaml

```diff
@@ -1168,6 +1302,9 @@ Outputs:
         - HasAlarmTopic
         - !GetAtt ErrorFatalLogsMetricAlarm.Arn
         - ''
+  CloudWatchAlarmOOMArn:
```

Comment:
@danilosilvase use the same logic of `FatalLogsMetricAlarmArn` to generate the output conditionally